src/main/java/chess/controller/command/End.java

```diff
@@ -0,0 +1,26 @@
+package chess.controller.command;
+
+import chess.domain.ChessGame;
+import chess.view.OutputView;
+
+public class End implements Command {
+
+    private final ChessGame chessGame;
+
+    public End(final ChessGame chessGame) {
+        this.chessGame = chessGame;
+    }
+
+    @Override
+    public void execute(final String rawInputCommand, final OutputView outputView) {
+        outputView.printFinishMessage();
+        if (chessGame.isNotRunning()) {
+            this.chessGame.off();
```

Comment:
```suggestion
            chessGame.off();
```
요즘엔 IDE가 잘 발달되어서 필요한 경우가 아니면 this는 잘 안 붙이는 추세입니다.

Reply:
생성자는 생략안되어서 그냥 두었습니다.! 
아직 자바에 익숙하지 않아서 this를 쓰는 것이 덜 헷갈리는데 이 부분은 고민해보겠습니다. 추세를 따라가는 개발자가 되고 싶습니다.ㅎㅎㅎ!
일단 버릇을 들이면 좋을 것 같아 `this.을 모두 삭제`하였습니다!

감사합니다 구구